README.md

```diff
@@ -26,6 +26,11 @@ You modify it until it becomes your own actual thesis:
 
 Your TeX distribution must include the appropriate packages for supporting the use of Hebrew in (Xe)LaTeX. Specifically, the Culmus fonts should be available, as one of them (David CLM) is used by default. In past versions, the template relied on the `babel` package and the `Jerusalem` and other old-style fonts; this is no longer the case: the [`polyglossia`](http://ctan.org/pkg/polyglossia) and the  [`bidi`](http://ctan.org/pkg/bidi) packages are now in use. Still, Hebrew is tricky and some packages have either compatibility issues with `polyglossia`/`bidi`, or must be loaded before `bidi`.
 
+### Overleaf support
+The package works seamlessly with overleaf by changing the compiler's setting to TexLive 2017:
```

Comment:
I'm pretty sure that's not the only compiler choice that works...

Reply:
I started my thesis last week.
I check most of the compilers and this was the only one that had no issues. Some compile well, but then had all kinds of issues such as showing numbers rtl instead of ltr.
Regardless, I think a comment regarding overleaf is in place since the default setting (TexLive 2020) doesn't work well.

Reply:
You're right, I suppose. I'll accept it for now and have another round of checking soon.

Reply:
I can confirm this - some compilation errors via the default setting. Typo in your note - templacte -> template

Reply:
> I can confirm this - some compilation errors via the default setting. Typo in your note - templacte -> template

Wait, is that a typo which causes errors somewhere? If so, please file a separate issue about this.

Also, I'm very sorry about the compilation difficulties. I'm waiting on the polyglossia guys to possibly address [the issue](https://github.com/reutenauer/polyglossia/issues/505) on their side. Alternatively, I will find some time to try converting the template to luatlatex+babel (not that there's any promise I won't encounter problems there as well).

About